pkg/render/applicationlayer/embed/embed.go

```diff
@@ -48,23 +48,27 @@ func AsMap() (map[string]string, error) {
 			return err
 		}
 
-		if b, err := fs.ReadFile(FS, path); err != nil {
+		if d.Name()[0] == '@' {
```

Comment:
not sure why this is done? are we skipping any directories prefixed with '@'? I thought rules were in a directory named '@owasp-' something

Reply:
Yes that works by accident, need to make the logic clearer.

Reply:
"by accident" i think you mean "very skilfully"

Reply:
Changing this turned out to be easier than anticipated :smile: 
![image](https://github.com/user-attachments/assets/668976ce-6c92-49f9-a907-933fec5d2a1c)